# origin: bk
# cset: 1.1525.4.2
# inclusion: n/a (projected: 2.4.28-pre3)
# description: spurious ampersand in spin_unlock_irqrestore call
# revision date: 2004-09-03

# This is a BitKeeper generated diff -Nru style patch.
#
# ChangeSet
#   2004/08/28 19:34:39-04:00 ha505@hszk.bme.hu 
#   [netdrvr fealnx] fix spin_unlock_irqrestore() usage
# 
# drivers/net/fealnx.c
#   2004/08/28 19:34:37-04:00 ha505@hszk.bme.hu +1 -1
#   [netdrvr fealnx] fix spin_unlock_irqrestore() usage
# 
diff -Nru a/drivers/net/fealnx.c b/drivers/net/fealnx.c
--- a/drivers/net/fealnx.c	2004-09-03 18:37:08 -07:00
+++ b/drivers/net/fealnx.c	2004-09-03 18:37:08 -07:00
@@ -1796,7 +1796,7 @@
 	unsigned long flags;
 	spin_lock_irqsave(lp, flags);
 	__set_rx_mode(dev);
-	spin_unlock_irqrestore(&lp, flags);
+	spin_unlock_irqrestore(lp, flags);
 }
 
 
